src/docs/build-and-earn/testnet-guides/run-full-node.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Run a Full Node
+---
+
+## ﻿Run a Full Node
+
+Before running a node, make sure that you have followed the guide for [setting up your prerequisites and environment](/docs/build-and-earn/testnet-guides/prerequisites.md).
+
+To run a full node, you need to complete the following steps:
+
+- Initialize the node
+- Configure and start the node
+
+## Initialize the node as a validator
+
+1. Navigate to the directory *CudosBuilders/docker/full-node*
+2. Find the file **full-node.env.example** and create a copy of it
+3. Rename the copied file to **full-node.client.testnet.public01.env**
```

Comment:
step 2 & 3 could be simply something like:
"copy **full-node.env.example** file to **full-node.client.testnet.public01.env**"

Reply:
I want to suggest keeping it as it is because I was confused while trying to make this step done.